cumulus/client/relay-chain-interface/src/lib.rs

```diff
@@ -363,4 +374,11 @@ where
 	async fn version(&self, relay_parent: PHash) -> RelayChainResult<RuntimeVersion> {
 		(**self).version(relay_parent).await
 	}
+
+	async fn claim_queue(
```

Comment:
With the introduction of this can we remove `availability_cores` from the interface? Did not check, but should not be used anymore now right?

Reply:
yes, we could. But is there a good reason for that? There's still information that the availability-cores API contains that you cannot get elsewhere: like when a core is occupied or not. Maybe this will come in handy at some point in the future.